Source/Core/DolphinQt/Config/ConfigDialog.h

```diff
@@ -0,0 +1,28 @@
+// Copyright 2015 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <memory>
+#include <QMainWindow>
+
+// Predefinitions
+namespace Ui
```

Comment:
Oh, actually, scratch that. This is a predefinition of a `uic`-generated class; so we can't change it.